app.controller.js

```diff
@@ -0,0 +1,15 @@
+const {fetchTopics} = require('./app.model')
+const endpoints = require('./endpoints.json')
+
+exports.getTopics = (req, res, next)=>{
+    fetchTopics().then((topics)=>{
+        if (!topics[0]) {
```

Comment:
I think this is a bit sad pathy, but I can see why you did it

Reply:
Not sure what you mean by this - should I be doing this in error handling perhaps?